trinity/nodes/base.py

```diff
@@ -46,6 +51,25 @@ def __init__(self, plugin_manager: PluginManager, trinity_config: TrinityConfig)
         self._headerdb = self._db_manager.get_headerdb()  # type: ignore
 
         self._jsonrpc_ipc_path: Path = trinity_config.jsonrpc_ipc_path
+        self._network_id = trinity_config.network_id
+
+        self.event_bus = plugin_manager.event_bus_endpoint
+
+    async def handle_network_id_requests(self) -> None:
+        async def f() -> None:
+            # FIXME: There must be a way to cancel event_bus.stream() when our token is triggered,
```

Comment:
nitpick: The comment could be redirected to mention that this hack will go away as soon as https://github.com/ethereum/lahja/pull/14 is included in the next release

Reply:
Will cleanup independently